.eslintrc.yml

```diff
@@ -13,7 +13,7 @@ plugins:
   - jest
 parser: '@typescript-eslint/parser'
 parserOptions:
-  project: ./tsconfig.json
+  project: ./tsconfig.eslint.json
```

Comment:
This is the same as `tsconfig.json` with the exception of `exclude: ['**/*.test.ts']` which fixes this error in test files:

```
Parsing error: ESLint was configured to run on `<tsconfigRootDir>/__tests__/main.test.ts` using `parserOptions.project`: <tsconfigRootDir>/../github-action-await-vercel/tsconfig.json
However, that TSConfig does not include this file.
```

(if you ever want to start running ESLint on the entire project)